.github/actions/polymorph_codegen/action.yml

```diff
@@ -88,18 +88,20 @@ runs:
         make -C mpl/AwsCryptographicMaterialProviders setup_prettier
         make -C mpl/ComAmazonawsKms setup_prettier
         make -C mpl/ComAmazonawsDynamodb setup_prettier
-
-    - name: Regenerate Java code using smithy-dafny
```

Comment:
(comment) I dislike anytime we have something in CI them remove it, but I understand this one